src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -114,51 +170,38 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
     transform: translateY($range-thumb-size / 5) scale(0.4);
   }
 
-  &::-webkit-slider-thumb {
-    margin-top: -($range-thumb-size - $range-track-height) / 2;
-  }
-
   .error & {
     --progress-lower: #{color('red')};
 
     @include range-track-selectors {
-      background-color: color('red', 'light');
+      background-color: var(--p-override-none, color('red', 'light'));
     }
 
     @include range-thumb-selectors {
-      border-color: color('red');
+      background: var(--p-critical, color('white'));
+      border-color: var(--p-critical, color('red'));
       box-shadow: $range-thumb-shadow-error;
     }
   }
 
   .disabled & {
     @include range-track-selectors {
       cursor: auto;
+      background-color: var(--p-border-on-inverse, color('sky', 'dark'));
       background-image: none;
     }
 
     @include range-thumb-selectors {
       cursor: not-allowed;
-      border-color: color('sky', 'dark');
-    }
-  }
-
-  // match `error` specificity for interaction styles
-  .RangeSlider &:focus {
-    // repeated so that `focus` can override `error`
-    --progress-lower: #{color('indigo')};
-
-    @include range-track-selectors {
-      background-color: color('sky', 'dark');
-    }
-
-    @include range-thumb-selectors {
-      background: linear-gradient(
-        color('sky', 'lighter'),
-        color('sky', 'light')
-      );
-      border-color: color('indigo');
-      box-shadow: $range-thumb-shadow-focus;
```

Comment:
This doesn't appear to be needed anymore.

Reply:
Can you add some more context to this?

Reply:
At first I removed it because I was unable to replicate the situation in which this is required. But since I don't fully understand why it was added in the first place, I'll keep it as is.